easybuild/easyconfigs/s/swak4Foam/swak4Foam-20190214-intel-2017a-OpenFOAM-4.1.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'ConfigureMake'
+
+name = 'swak4Foam'
+version = '20190214'
+commit = 'e1d2358'
```

Comment:
@smoors Any reason you're not using one of the releases? https://github.com/Unofficial-Extend-Project-Mirror/openfoam-extend-swak4Foam-dev/releases

Reply:
As written here: https://openfoamwiki.net/index.php/Installation/swak4Foam/Downloading#OpenFOAM_4.0_to_4.x you need the development version for OF4.X support.
Upon closer look there are indeed new releases out that do support more recent versions of OF.
Quite confusing though: a release of a development version...